chrome_manifest_v3/content-script.js

```
@@ -48,7 +84,7 @@ function g2t_log(data) {
       data = JSON.stringify(data);
     }
 
-    data = now_k + '.' + counter_k + ' G2T→' + data;
+    data = `${now_k}.${counter_k} G2T→${data}`;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)